rows.go

```diff
@@ -700,6 +702,7 @@ func (f *File) InsertRows(sheet string, row, n int) error {
 // worksheet, it will cause a file error when you open it. The excelize only
 // partially updates these references currently.
 func (f *File) DuplicateRow(sheet string, row int) error {
```

Comment:
For the `DuplicateRow`, `InsertCol` and `InsertRow` functions, we can clear calc cache in `f.adjustHelper` function instead of call `f.clearCalcCache` on each functions.